test/tic_tac_toe/user_interface_test.clj

```diff
@@ -21,6 +21,12 @@
   (testing "receives a message and displays it"
     (is (= "Hello" @(:output test-io)))))
 
+(deftest checks-input-for-esc
```

Comment:
It looks like there is not a test for when the input is `nil`. Is there a way to test that?

Reply:
[Adjusted and added test](https://github.com/saralein/tic-tac-toe_clojure/pull/12/files#diff-c76be95098989bdafb535fcf2a85dc18R26).  Thank you.

Reply:
Given the other PR, I don't think this is an appropriate solution for this issue.  I will explore my options.

Reply:
OK, [modified the UI a bit](https://github.com/saralein/tic-tac-toe_clojure/pull/12/files#diff-52c69c82bcdbcc16e3156d4a9033a37cR10).  On the plus side, this allows me to [test nil input and test the exit method](https://github.com/saralein/tic-tac-toe_clojure/pull/12/files#diff-c76be95098989bdafb535fcf2a85dc18R27).  On the negative side, I had to do [this](https://github.com/saralein/tic-tac-toe_clojure/pull/12/files#diff-15f263ec5b23d4d6b199b76016540567R37).  But I tried to make it a little flexible at least by not hard typing the class.